src/nginxconfig/scss/_panel.scss

```
@@ -62,8 +62,13 @@ THE SOFTWARE.
       align-items: center;
 
       .button {
+        flex: auto;
         margin: 0 .25rem .5rem;
       }
+      
+      .button:last-child {
+        flex: 0;
```

```suggestion
      
        &:last-child {
          flex: 0;
        }
```
This is scss, so selectors should be nested